net.sonic_pi.SonicPi.json

```diff
@@ -154,10 +165,13 @@
             "name": "sonicpi",
             "buildsystem": "simple",
             "build-commands": [
-                "cd app && ./linux-prebuild.sh --build-aubio",
-                "cd app && ./linux-config.sh --config RelWithDebInfo",
-                "cmake --build app/build"
+                "cd app && ./linux-build-all.sh --config RelWithDebInfo && ./linux-release.sh"
```

Comment:
From what I know, the whole build process has been revamped and this simplifies this section, it simply works with the provided scripts (I left the `--config RelWithDebInfo` option around; I don't know exactly what it does, but I checked that it gets passed through).